extensions/amp-animation/amp-animation.md

```diff
@@ -551,17 +365,17 @@ Both `var()` and `calc()` polyfilled on platforms that do not directly support t
 </amp-animation>
 ```
 
-Animation components can specify their own variables as `--var-name` fields. These variables are propagated into nested animations and override variables of target elements specified via stylesheet (`<style>` tag). `var()` expressions first try to resolve variable values specified in the animations and then by querying target styles.
+Override variables of target elements specified in the `<style amp-custom>` tag by using `--var-name` fields in `amp-animation` component. `var()` expression first try to resolve values specified within the animation component and then query target styles.
```

Comment:
Let's change "query target styles" to "resolve target styles". Querying is used a lot in the relationship to selectors in this doc.